assets/src/blocks/Happypoint/HappypointEditor.js

```diff
@@ -38,203 +38,202 @@ export const HappypointEditor = ({ attributes, setAttributes, isSelected }) => {
+  const [ iframeUrl, setIframeUrl ] = useState( iframe_url || '' );
+  const [ embedCode, setEmbedCode ] = useState( embed_code || '' );
   const dimensions = { width: 400, height: 100 };
 
-  const { imageUrl } = useSelect(select => {
-    let imageUrl = '';
-    if (id && id > 0) {
-      const imageDetails = select('core').getMedia(id);
-      imageUrl = (imageDetails && imageDetails.source_url) || '';
+  const { imageUrl } = useSelect( ( select ) => {
+    let imageUrlSelect = '';
+    if ( id && id > 0 ) {
+      const imageDetails = select( 'core' ).getMedia( id );
+      imageUrlSelect = ( imageDetails && imageDetails.source_url ) || '';
     }
-    return { imageUrl };
-  }, [id]);
+    return { imageUrl: imageUrlSelect };
+  }, [ id ] );
 
   let focal_point_params = { x: '', y: '' };
 
-  if (focus_image) {
-    let focus_image_str = focus_image.replace(/%/g, '');
-    let [x, y] = focus_image_str.split(' ');
+  if ( focus_image ) {
+    const focus_image_str = focus_image.replace( /%/g, '' );
+    const [ x, y ] = focus_image_str.split( ' ' );
     focal_point_params = { x: x / 100, y: y / 100 };
   } else {
     focal_point_params = { x: 0.5, y: 0.5 };
   }
 
-  const getImageOrButton = openEvent => {
-    if (id && 0 < id) {
-      return <HappypointFrontend {...attributes} />;
-    } else if (isSelected) {
+  const getImageOrButton = ( openEvent ) => {
+    if ( id && 0 < id ) {
+      return <HappypointFrontend { ...attributes } />;
+    } else if ( isSelected ) {
       return (
-        <div style={{ marginBottom: 10 }}>
+        <div style={ { marginBottom: 10 } }>
           <Button
-            onClick={openEvent}
-            className='button'>
-            + {__('Select Background Image', 'planet4-blocks-backend')}
+            onClick={ openEvent }
+            className="button">
+						+ { __( 'Select Background Image', 'planet4-blocks-backend' ) }
           </Button>
         </div>
       );
     }
     return null;
   };
 
-  const toAttribute = attributeName => value => setAttributes({
-    [attributeName]: value
-  });
+  const toAttribute = ( attributeName ) => ( value ) => setAttributes( {
+    [ attributeName ]: value,
+  } );
 
-  const onFocalPointChange = ({ x, y }) => {
-    const floatX = parseFloat(x).toFixed(2);
-    const floatY = parseFloat(y).toFixed(2);
-    setAttributes({ focus_image: `${floatX * 100}% ${floatY * 100}%` });
-  }
+  const onFocalPointChange = ( { x, y } ) => {
+    const floatX = parseFloat( x ).toFixed( 2 );
+    const floatY = parseFloat( y ).toFixed( 2 );
+    setAttributes( { focus_image: `${ floatX * 100 }% ${ floatY * 100 }%` } );
+  };
 
-  const onRemoveImages = () => setAttributes({ id: '', focus_image: '' });
+  const onRemoveImages = () => setAttributes( { id: '', focus_image: '' } );
 
-  const selectImage = ({ id }) => setAttributes({ id });
+  const selectImage = ( { imageId } ) => setAttributes( { imageId } );
```

Comment:
Here this will not work, it should be something like 
```
const selectImage = ( { id: imageId } ) => setAttributes( { id: imageId } );
```
or 
```
const selectImage = ( selectedImage ) => setAttributes( { id: selectedImage.id } );
```

Reply:
Done